scripts/pylint_extensions.py

```diff
@@ -706,10 +672,7 @@ def visit_importfrom(self, node):
         except astroid.AstroidBuildingException:
             return
 
-        if node.level is None:
-            modname = node.modname
-        else:
-            modname = '.' * node.level + node.modname
```

Comment:
We cannot relatively import modules with our current codebase hence this line is unreachable

Reply:
Hmm, but why delete it? You can test it, right? We might use relative import in the future to remove pylint pragmas. We even have an issue filed for this: https://github.com/oppia/oppia/issues/5714.

Reply:
I get an error like this: `ValueError: Attempted relative import in non-package`

Reply:
When are you getting this error? This occurs when pylint pragmas, which disable relative-import, are removed. Is this occurring when you're trying to test this? How are you trying to test this?

Reply:
No that's when I import it in the file. I am getting an error like: `TooManyLevelsError: Relative import with too many levels (1) for module ''` when I use it in `astroid.extract_node`

I searched this code in Pylint but did not find anything. Can you tell from where you derived this code?

Reply:
When you say "import it in the file", is it just a simple import like `import math` or is it a deeper level import?

Reply:
No its a relative import like `from . import build`

Reply:
Hmm, can you find out a way to perform relative imports then? We'll need do this in the future.

Reply:
Done